REST/src/main/java/pl/jakubowskiprzemyslaw/REST/utils/SendMessage.java

```diff
@@ -0,0 +1,18 @@
+package pl.jakubowskiprzemyslaw.REST.utils;
+
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SendMessage {
+
+    private final SimpMessagingTemplate template;
+
+    public SendMessage(SimpMessagingTemplate template) {
+        this.template = template;
+    }
+
+    public void sendMessage(String playerName, String message) {
```

Comment:
mention about clock

Reply:
it's done already, will be shown in next PR